bin/run.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+die() { echo "$*" >&2; exit 1; }
+
+usage() {
+    local usage="Usage: ${0##*/} slug inputDir outputDir"
+    (($#)) && printf -v usage '%s\n%s' "$*" "$usage"
```

Comment:
You're using `printf` to write to usage ... and read from usage? I find this function confusing to read.

Reply:
Woops